src/controllers/accommodation.js

```diff
@@ -215,6 +215,42 @@ class accommodation {
       });
     }
   }
+
+  /**
+   * 
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} responsd
+   */
+  static async getOneAccommodation(req, res) {
+    try {
+      const {
+        params: {
+          id
+        },
+        user: {
+          preferedLang
+        }
+      } = req;
+      const result = await Accommodations.findOne({
+        where: { id },
+        include: [{
+          model: Rooms,
+          attributes: {
+            exclude: ['accommodationsID', 'createdBy', 'createdAt', 'updatedAt']
+          }
+        }]
+      });
```

Comment:
Please if possible include feedback too

Reply:
@blaise82  it is possible but because you guys had not defined those relationships, am supposed to go back into models and define them.

Reply:
@blaise82 but now it's done, feedbacks are returned and the name of the one who gave the feedback too.

Reply:
thanks